src/ecs/rendering/resources.rs

```diff
@@ -4,7 +4,7 @@ use std::sync::Arc;
 
 use crossbeam::sync::MsQueue;
 use futures::{Async, Future, Poll};
-use futures::sync::oneshot::{Receiver, Sender, channel};
+use futures::sync::oneshot::{channel, Receiver, Sender};
```

Comment:
Why has this been reordered?

Reply:
rustfmt

Reply:
I think we agreed on having `Type`s before `function`s.

cc @Rhuagh

Reply:
Honestly I'm so over burning development time on how to sort <5 item import lists.